packages/ag-charts-community/src/chart/toolbar/toolbar.ts

```diff
@@ -131,6 +136,7 @@ export class Toolbar extends BaseModuleInstance implements ModuleInstance {
             ctx.toolbarManager.addListener('button-toggled', this.onButtonToggled.bind(this)),
             ctx.toolbarManager.addListener('button-updated', this.onButtonUpdated.bind(this)),
             ctx.toolbarManager.addListener('group-toggled', this.onGroupToggled.bind(this)),
+            ctx.toolbarManager.addListener('group-toggled', this.onGroupUpdated.bind(this)),
```

Comment:
Should this be listening to the `group-updated` event?

Reply:
Correct. Well spotted! I'll change it and re-test.

Reply:
Done.